### DISCUSSION TITLE
Query Regarding Special Character Input in Content Creation Title

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Lern/Community#184] [Query]
Hello Team,

I am currently encountering an issue while attempting to input special characters into the title field during content creation. It appears that the system does not allow the inclusion of certain special characters.

Could someone kindly provide guidance on whether it is possible to allow the input of special characters in the title field? If so, what steps can be taken to enable this functionality?

I appreciate any assistance or insights that you can offer regarding this matter. Thank you for your time and attention.

**Comment by AmiableAnil** [comment]
@pallakartheekreddy can you please have a look into it?

cc: @PoojaJManjooran @mohitga
